census/estimator/trainer/model.py

```diff
@@ -215,29 +214,20 @@ def example_serving_input_fn():
       example_bytestring,
       tf.feature_column.make_parse_example_spec(INPUT_COLUMNS)
   )
-  features = {
-      key: tf.expand_dims(tensor, -1)
-      for key, tensor in feature_scalars.iteritems()
-  }
-  return tf.contrib.learn.InputFnOps(
+  return tf.estimator.export.ServingInputReceiver(
       features,
-      None,  # labels
       {'example_proto': example_bytestring}
```

Comment:
This needs to be 'example_bytes'

Reply:
Also, untested, but I wonder if this can be 'build_parsing_serving_input_receiver_fn'

Reply:
Similar comment as above on the build...() API.  But receiver_tensor is a tensor dict so it has to be a Tensor.